.eslintrc.js

```diff
@@ -3,4 +3,5 @@ module.exports = {
   'ecmaFeatures': {
     'modules': true,
   },
+  'parser': 'babel-eslint',
```

Comment:
I'm a little confused, arent' we _only_ using ES2015? In which case we shouldn't need `babel-eslint`:
> Note: You don't need to use babel-eslint if you are only using ES6/ES2015

I'm happy that you updated eslint dependencies. We've been behind for a while.

If you want to make it easier for me to merge the PR, you should try to avoid introducing other changes. I think this is fine however, so don't worry. :)